net/pfSense-pkg-haproxy/files/usr/local/www/haproxy/haproxy_global.php

```diff
@@ -135,6 +135,7 @@
 					unset($haproxycfg['config'][0]['enable']);
 				}
 			} else {
+				$haproxycfg['config'][0] = array();
```

Comment:
That would help if just the one level is missing but a better fix is to not add this line, but to use the new access functions to read the configuration variable below. I'll post a follow-up comment on that line.